src/fiskaltrust.Launcher.Common/Configuration/Configuration.cs

```diff
@@ -27,29 +27,30 @@ public record LauncherConfiguration
     {
         public const string DATA_PROTECTION_DATA_PURPOSE = "fiskaltrust.Launcher.Configuration";
 
-        private bool _useDefaults;
+        private bool _raw;
```

Comment:
Not 100% sure what this field does, tbh 😅

Reply:
is it clearer now?